src/text.rs

```diff
@@ -195,7 +198,7 @@ impl<'a> From<&'a str> for Span<'a> {
 
 /// A string composed of clusters of graphemes, each with their own style.
 #[derive(Debug, Clone, PartialEq, Default, Eq)]
-pub struct Spans<'a>(pub Vec<Span<'a>>);
+pub struct Spans<'a>(pub Vec<Span<'a>>, pub Option<Alignment>);
```

Comment:
I'm not 100% certain myself on the right implementation for this (I'm fairly new to rust/tui myself) I agree with your logic that it's probably not a major breaking change (consumers are probably calling `Spans::from()` and that's a reasonably easy automated change rather than having to add the None param everywhere.

On thinking about this more, having the concept of alignment within a span seems more reasonable than specifically making this a new type. In particular if it's a new type then any future behavior can't easily make a third type that support no alignment vs being aligned.

I think it could be worth however going from `struct Spans(vec, alignment)` to `struct Spans { spans: Vec<Span>, alignment: Alignment }` perhaps to avoid the tuple readabilty tax (what is `.0` / `.1`).